### ISSUE TITLE
training data

### ISSUE THREAD

**Issue by Shu-HowTing** [weders/DeepDescriptors#1]
Hi, guys, nice to read your codes!
If what I thought is right, you are recurring this paper`SuperPoint: Self-Supervised Interest Point Detection and Description`. I just woder if  you had the training data?If you do have ,can you share it with me? thank you!
Best wishes!

**Comment by noahho** [comment]
Hi Shu,
you're right, we are working on the paper you mentioned. For making that work you just need any general image database as training data. They are using MS COCO (http://cocodataset.org/#home) and the algorithm will construct the keypoint labels in a first step.
Take a look at this code: https://github.com/rpautrat/SuperPoint/tree/master/notebooks . They have already implemented a lot of the code and you could maybe run it to generate labels.

**Comment by Shu-HowTing** [comment]
Hi，
thank you for your reply.I will have a try , and I am focus on solving end-to-end loacal descriptors learning(detect+descriptor) instand of patch ways. I have been troubled by this problem for a long time.
Anyway,thanks again!






On 12/18/2018 02:03，noahho<notifications@github.com> wrote：

Hi Shu,
you're right, we are working on the paper you mentioned. For making that work you just need any general image database as training data. They are using MS COCO (http://cocodataset.org/#home) and the algorithm will construct the keypoint labels in a first step.
Take a look at this code: https://github.com/rpautrat/SuperPoint/tree/master/notebooks . They have already implemented a lot of the code and you could maybe run it to generate labels.
